airbyte-integrations/connectors/source-monday/unit_tests/test_graphql_requester.py

```diff
@@ -53,8 +53,9 @@
             nested_array_schema,
             "items",
             {},
-            {"query": "query{boards(limit:100,page:2){items(limit:100,page:1){root{nested{nested_of_nested}},sibling}}}"},
-            {"next_page_token": (2, 1)},
+            # {"query": "query{boards(limit:100,page:2){items(limit:100,page:1){root{nested{nested_of_nested}},sibling}}}"},
```

Comment:
What is the use for this commented line?

Reply:
removed